Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIBookmarksCocoa.mm

```diff
@@ -63,7 +74,14 @@
 
 static NSDictionary *toWebAPI(const WebExtensionAPIBookmarks::MockBookmarkNode& node)
 {
-    auto tempNode = @{
+    NSMutableArray* childrenArray = nil;
```

Comment:
`NSMutableArray *childrenArray = [NSMutableArray array];
i think you can initialize it here, and then set the childrenArray below if this isn't empty